index.html

```diff
@@ -151,1145 +151,1103 @@ <h4>Other Related Resources</h4>
       </section>
 
 
-      <section id="h_persian_language">
-        <h4>Persian Language</h4>
-
+      <section id="h_persian">
+        <h4>Persian</h4>
 
         <p><dfn>Persian</dfn>—a.k.a. Modern Persian—is the standardized and literary variety of the
         official languages used in Iran and Afghanistan. The dialect of Persian in Iran is also
         called Western Persian, and is locally known as <span class="qterm">Farsi</span>. The
         dialect of Persian in Afghanistan is also known as Eastern Persian, and is locally known as
         Dari.</p>
 
-
         <p><dfn>Tajik</dfn>—a.k.a Tajiki or Tajiki Persian—is the Persian language as used in
         Tajikistan. It is written in the Cyrillic script, therefore, is <em>not</em> covered by
         this document.</p>
       </section>
     </section>
-  </section>
 
 
-  <section id="h_arabic_script_overview">
-    <h2>Arabic Script Overview</h2>
+<section id="h_the_islamic_manuscript_tradition">
+        <h3>The Islamic manuscript tradition</h3>
 
+        <p>Islamic manuscript practice evolved with Islamic art and civilization. Extant evidence of decorative use of the Arabic script can be found as early as the 7th century CE, in a mosaic band in the Dome of the Rock. Beyond frequent monumental inscriptions in architecture, calligraphy became one of the foremost Islamic art forms. Manuscript practice evolved into a central element of Islamic civilization through the copying of books, and the refinement of calligraphy into a form of fine art. The commissioning and writing of the Qur‘an was seen as an expression of religious devotion and provided the context for scribes and calligraphers to excel in their craft. Further to its use as a textual medium, Arabic calligraphy and lettering crafts developed a rich palette of decorative forms and uses.</p>
+    <section id="writing_styles">
+        <h4>Writing Styles</h4>
+        <p>A range of styles evolved over time in different regions and for different purposes. These include monumental writing styles used in decorative and representational settings, styles used for the exclusive use of rulers, common hands in different areas, and styles dedicated for specialist purposes such as miniature writing. Many of these styles fell out of use, some evolved further, and some retained a high degree of consistency over time.</p>
+        <p>There is evidence of both, monumental and current hands to have existed and evolved from the early Islamic era onwards. Various exponents of the monumental styles are often loosely referred to as Kūfī, making it a term that lacks precision and clarity. It describes a group of styles that are marked by an angularity and stark graphic charcteristics that are absent from the round scripts that took preeminence from the late 10th, early 11th century CE.</p>
```

Comment:
Perhaps reword the first sentence as: There is evidence that both monumental and current hands existed and evolved from the early Islamic era onwards.